themes/portzero/layouts/partners/list.html

```diff
@@ -5,8 +5,7 @@ <h2 class="tagline">{{ .Title }}</h2>
 <div class="List">
   {{ range where .Data.Pages "Type" "=" "partner" }}
   <article class="List-item">
-    {{ with.Params.image }} <img src="{{ . }}" class="List-image" /> {{ else }}
-    <img src="img/logo.png" class="List-image" /> {{ end }}
+    <img alt="{{ .Title }} logo" src="{{ with .Params.image }}{{ . }}{{ else }}img/logo.png{{ end }}" class="List-image" />
```

Comment:
```suggestion
    <img alt="{{ .Title }} logo" src="{{ .Params.image | default "img/logo.png" }}" class="List-image" />
```

maybe use the default helper to avoid all the brackets? other lgtm

Reply:
Oh, I didn’t know about that! Cool!

Reply:
The alt texts use this construct now.